FrontEnd/src/components/AboutPage/MeetTheTeam/MemberInfoCard/MemberInfoCard.module.scss

```diff
@@ -0,0 +1,78 @@
+$outerOrangeBorder: #f8e6d8;
```

Comment:
The color "f8e6d8" is already in a variable. Take a look here: https://github.com/bamit5/Housing_Proj/blob/master/FrontEnd/src/assets/scss/utils/_global-vars.scss
and look for $border-color